codes/quantum/qubits/surface/triangle.yml

```diff
@@ -0,0 +1,56 @@
+code_id: triangle
+physical: qubits
+logical: qubits
+
+name: 'Triangle code'
+introduced: '\cite{doi:10.22331/q-2017-04-25-2}'
+
+description: 'Triangle codes are types of non-CSS topological quantum error-correction codes with modified twist defects. A twist is a check of weight five that is embedded into the lattice of the code. For the triangle codes, the modified twist defects have weight four checks. Additionally, the codes have simple syndrome extraction circuits nad have a high threshold close to that of surface codes. Triangle codes also use about \(25\%\) fewer physical per logical qubit compared to the surface code \cite{doi:10.22331/q-2017-04-25-2}.'
+
+protection: 'Protects against depolarizing noise. In particular, with \(13\) physical qubits triangle codes have a high-pseudothreshold fault-tolerance for depolarizing noise \cite{doi:10.22331/q-2017-04-25-2}.'
```

Comment:
Is there a [[n,k,d]] notation for this code? If so, please list here.

Reply:
There was not one provided, sadly.